### ISSUE TITLE
Default model should be simplified and use the built-in PyTorch transformer layers (though still support structured attention)

### ISSUE THREAD

**Issue by mmcdermott** [mmcdermott/EventStreamGPT#10]

**Comment by mmcdermott** [comment]
Specifically, this: https://pytorch.org/docs/stable/generated/torch.nn.functional.scaled_dot_product_attention.html?highlight=scaled_dot_product_attention#torch.nn.functional.scaled_dot_product_attention

**Comment by mmcdermott** [comment]
Closing as not planned due to upcoming deprecation. There are also better ways to do this these days, including using more modern transformers libraries and eliminating padding altogether, etc.
